sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2/async_samples/sample_analyze_business_cards_async.py

```diff
@@ -160,4 +160,13 @@ async def main():
 
 
 if __name__ == "__main__":
-    asyncio.run(main())
+    import sys
```

Comment:
Thinking about having a comment here that says something like: 
"Example for how to catch specific errors"...or something like that.